app/forms/tagging_update_form.rb

```diff
@@ -2,27 +2,38 @@ class TaggingUpdateForm
   include ActiveModel::Model
   attr_accessor :content_id, :previous_version
 
-  attr_accessor(*ContentItemExpandedLinks::TAG_TYPES)
+  TAG_TYPES = ContentItemExpandedLinks::TAG_TYPES
+  attr_accessor(*TAG_TYPES)
 
   # The number of extra empty form fields to add to a link section when the link
   # section shows an individual form input for each value. This allows users to
   # append new links to the end of the existing list.
   EXTRA_TEXT_FIELD_COUNT = 5
 
-  validate :related_item_paths_should_be_valid
+  if TAG_TYPES.include?(:ordered_related_items)
+    validate :related_item_paths_should_be_valid
+  end
 
   # Return a new LinkUpdate object with topics, mainstream_browse_pages,
   # organisations and content_item set.
   def self.from_content_item_links(content_item_links)
+    tags = TAG_TYPES.each_with_object({}) do |tag_type, current_tags|
+      if tag_type == :ordered_related_items
```

Comment:
Yes, that's correct. We would need to add the same conditional as above to the end of that line:

```
if TAG_TYPES.include?(:ordered_related_items)
  tags[:ordered_related_items] =
     pad_with_empty_items(tags[:ordered_related_items])
end
```

I think keeping it separate makes it easier to read, but also happy to leave it as is.